google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/scaladsl/GooglePubSub.scala

```diff
@@ -127,15 +140,28 @@ object GooglePubSub {
       .mapMaterializedValue(_.flatMap(identity)(ExecutionContexts.parasitic))
   }
 
-  private def publisher(mat: Materializer, attr: Attributes) =
+  @ApiMayChange
```

Comment:
These are not supposed to be public user facing API? Then they should be marked with `@InternalApi` instead.

Reply:
I think they should be left private, or if I'm missing a use case, please describe it a bit.